docs/current_docs/configuration/modules.mdx

```diff
@@ -74,16 +73,53 @@ You can also set a specific version with a suffix (e.g., `bun@1.0.11`).
   }
 ```
 
+Alternatively, executing `bun init` or `bun install` will create a `bun.lock` or `bun.lockb` file, which Dagger will [automatically detect](#automatic-detection) and configure Bun for use:
+
+```shell
+bun init
+dagger init --sdk=typescript --name=example --source=.
+```
+
+:::important
+The module's `source` field must reference the same directory as the `bun.lock` or `bun.lockb` file.
+:::
+
 :::warning
 Bun runtime support is still experimental. Unexpected results may occur in some cases.
 :::
+</TabItem>
+<TabItem value="Deno">
+Set the field to `deno` to use the Deno runtime.
+
+```json
+  "dagger": {
+    "runtime": "deno"
```

Comment:
This is actually the opposite, you can set `dagger.runtime=Bun` but not `dagger.runtime=Deno` because there's no package.json in Deno

Reply:
Updated